src/calculator.jsx

```diff
@@ -1,7 +1,11 @@
-import React from "react";
+import React, { Component } from "react";
 
-export default class Calculator extends React.Component {
+export default class Calculator extends Component {
     render() {
         return <div>I'm a calculator</div>;
     }
-}
\ No newline at end of file
+}
+
+export const FunctionCalculator = (props) => (
+	<div>I'm a calculator</div>
+)
```

Comment:
cool thing about components that don't need lifecycle methods and don't have state - they can just be a function!